tests/test_lib_token.py

```diff
@@ -1771,3 +1773,172 @@ def test_07_reset_failcounter_on_pin_only(self):
         delete_privacyidea_config(SYSCONF.RESET_FAILCOUNTER_ON_PIN_ONLY)
         remove_token("test07")
 
+
+class PINChangeTestCase(MyTestCase):
+    """
+    Test the check_token_list from lib.token on an interface level
+    """
+
+    def test_00_create_realms(self):
```

Comment:
I think it makes more sense to put this into a setup/teardown hook since it is required by all test functions. That way they can run independently.

Reply:
In theory you are right if you would run single test methods separately. This way we also save (some?) time. 

In fact we do this in a lot of cases in many tests.